helm_deploy/cmd-api/templates/cronjob-shift-changes-refresh.yaml

```diff
@@ -4,7 +4,7 @@ kind: CronJob
 metadata:
   name: "refresh-shift-changes"
 spec:
-  schedule: "0 0,3,9,15,21 * * *"
+  schedule: "10 0-21/3 * * *"
   concurrencyPolicy: Allow
```

Comment:
Move this to 10 past to not clash with the send job, run every 3 hours.

Reply:
does */3 mean every 3 hours from 12?

Reply:
it means every 3 hours  24/7 
https://crontab.guru/#10_*/3_*_*_*

Reply:
pulling the notification data this much is fine, its the excessive sending of emails and texts we wanted to address.

Reply:
Why 10 minutes for the stagger? with the last notification going out at 18:00, thats gonna be getting data up to 15:10? Could/should we stagger these to be within half an hour maybe? Or have I misunderstood this?

Reply:
Sending happens on the hour, and takes about 5 mins, they can run at the same time problem, but I don't want them to run at the same time because that will take resources away from serving users.

Reply:
> thats gonna be getting data up to 15:10
it runs at to past 0, 3, 6, 9, 12, 15, 18, 21
we send whatever we have out at 
6, 10, 14, 18

Reply:
Am I right in thinking that:
- sending refreshes notifications before they're sent so the cron jobs are not dependent on each other
- refresh is just for notifications tab on the calendar app?

If so it might be nice to stagger out the refresh to half past? Although tbh I am confused why we are sending emails out more than once a day. I personally feel like an end of day roundup at 6pm makes more sense 🤷🏽‍♀️

Reply:
Right, these are now at half past.
We need to cater to night and day shift workers, so one in the morning and one in the afternoon makes sense?
I've changed it to send at 9 and 18 only.